x-pack/plugins/rule_registry/server/search_strategy/search_strategy.ts

```diff
@@ -146,6 +146,8 @@ export const ruleRegistrySearchStrategyProvider = (
         map((response) => {
           // Do we have to loop over each hit? Yes.
           // ecs auditLogger requires that we log each alert independently
+          //
```

Comment:
These 2 added lines are the only reason that a review is requested from the actionable observability team, shall we remove them?
Then there is no need for code-owner review from the actionable observability team.

Reply:
@maryam-saeidi , @done. Thank you.